src/cli.rs

```diff
@@ -397,6 +404,22 @@ fn parse_bezier(raw: &str) -> Result<(f32, f32, f32, f32), String> {
     Ok(parsed)
 }
 
+pub fn parse_image(raw: &str) -> Result<CliImage, String> {
+    if raw.starts_with("0x") {
+        let color = from_hex(raw.strip_prefix("0x").unwrap());
+        if let Ok(color) = color {
+            return Ok(CliImage::Color(color));
+        }
+    }
+
+    let path = PathBuf::from(raw);
+    if !path.exists() {
+        return Err(format!("Path '{}' does not exist", raw));
+    }
+
+    Ok(CliImage::Path(path))
+}
```

Comment:
I would change the order of these so that if someone has a file which starts with `0x` the file will be picked
```suggestion
    let path = PathBuf::from(raw);
    if path.exists() {
        return Ok(CliImage::Path(path));
    }
    if let Some(color) = raw.strip_prefix("0x") {
        if let Ok(color) = from_hex(color) {
            return Ok(CliImage::Color(color));
        }
    }
    Err(format!("Path '{}' does not exist", raw))
}
```